infra.bs

```diff
@@ -1785,21 +1785,29 @@ entries with a comma.
 <hr>
 
 <p>To <dfn export for=map lt="get|get the value">get the value of an entry</dfn> in an
-<a>ordered map</a> <var>map</var> given a <a for=map>key</a> <var>key</var>:
+<a>ordered map</a> <var>map</var> given a <a for=map>key</a> <var>key</var> and an optional
+<var>default</var>:
```

Comment:
This is missing a type declaration for _default_.

Reply:
I don't think we necessarily need one, since it can be any value.

Reply:
Not sure if this is what is meant, but I've now declared it as a map/value, since that corresponds to the role that |default| takes on here.

Reply:
I guess that works, although those are not really types and maybe you want something outside of what the tuple can contain. Hmm.

I briefly thought of defining "any/anything" but it will look so ugly in prose. Slowly coming around to @domenic's side of leaving the type out when it can be anything, even if it's a bit ambiguous. @domenic any final thoughts?

Reply:
I think leaving the type out is best.

Reply:
I removed the type again.